### PR TITLE
 feat(overlay-directives): support setting panelClass

### PR DESCRIPTION
**Pull request by jianliao**
Adds the ability for customizing the panelClass of overlay.

### CODE REVIEW

File: src/demo-app/table/custom-table/wrapper-table.ts [hunk 1]
```diff
@@ -45,10 +45,22 @@ export class WrapperTable<T> implements AfterContentInit {
 
   @ViewChild(MatTable) table: MatTable<T>;
 
-  @Input() columns: string[];
+  private _columns: string[];
+
+  // @Input() columns: string[];
```
[the remark this answers is not in the document]
Comment (jianliao) [reply src/demo-app/table/custom-table/wrapper-table.ts]: Yes, sorry. My local change leak to this PR branch.

**Comment by jianliao** [comment]
I think I signed the CLA and confirmed from https://cla.developers.google.com/.

**Comment by crisbeto** [comment]
You have to respond with a comment that says "I signed it!".

**Comment by jianliao** [comment]
I signed it!

**Comment by crisbeto** [comment]
If that didn't work, it means that you might've signed the CLA with a different email address than you've configured in git. Note how a couple of the PR commits show a different author.

**Comment by jianliao** [comment]
ok, let me close this pr and do it again.

**Pull request closed without merging**